aws/viewer-request/index.js

```diff
@@ -113,10 +113,13 @@ function handler(event) {
 			return redirect('https://surrealdb.com/docs/surrealdb/faqs');
 		case '/docs/how-to/overview':
 		case '/docs/surrealdb/how-to/overview':
-			return redirect('https://surrealdb.com/docs/surrealdb/tutorials');
+			return redirect('https://surrealdb.com/docs/tutorials');
 		case '/docs/tutorials/overview':
 		case '/docs/surrealdb/tutorials/overview':
-			return redirect('https://surrealdb.com/docs/surrealdb/tutorials');
+			return redirect('https://surrealdb.com/docs/tutorials');
+		case '/docs/surrealdb/tutorials':
+		case '/docs/surrealdb/tutorials/':
```

Comment:
We don't have this for the others, do we need it?

Reply:
We don't need it per say I'll remove it